Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxCode.bind

```diff
@@ -1,25 +1,95 @@
-private void TextChanged(AutoSuggestBox sender, AutoSuggestBoxTextChangedEventArgs args)
-{
-    if (args.CheckCurrent() && args.Reason == AutoSuggestionBoxTextChangeReason.UserInput)
-    {
-        if (string.IsNullOrWhiteSpace(sender.Text))
+
+        private void TextChanged(AutoSuggestBox sender, AutoSuggestBoxTextChangedEventArgs args)
```

Comment:
Is this extra tabbing one the bind file intended?

Reply:
hmm, not sure. I've removed it all